cmd/minikube/cmd/docker-env.go

```diff
@@ -245,6 +246,7 @@ var dockerEnvCmd = &cobra.Command{
 		shl := shell.ForceShell
 		if shl == "" {
 			shl, err = shell.Detect()
+			shl = strings.TrimSuffix(shl, filepath.Ext(shl))
```

Comment:
Would be a little bit better to first check the error and then if it's nil trim the '.exe'